src/cap-description.adoc

```diff
@@ -93,36 +93,53 @@ Therefore, it is only possible to encode a subset of all combinations.
 |==============================================================================
```

Comment:
This table and the previous paragraph state that the RV32 encoding is 4 bits although it is 5 bits with this proposed encoding.